src/Bicep.Decompiler/TemplateConverter.cs

```diff
@@ -1618,6 +1631,8 @@ private ProgramSyntax Parse()
                 }
             }
 
+                    // We are ignoring _generator metadata since it is generated by bicep build
+            AddSyntaxBlock(statements, metadata.Where(x => x.Name != "_generator").Select(ParseMetadata), false);
```

Comment:
Should we block all metadata starting with `_`? IIRC the plan was to consider this 'system reserved' and prevent it from being used in Bicep.

Reply:
This is my understanding as well. Does this work for you @simonwahlin?